deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinCollection.java

```diff
@@ -477,6 +489,14 @@ private void fillJsonMetadata(JsonObject jsonMetadata)
         {
             jsonMetadata.addProperty(TwinMetadata.LAST_UPDATE_TAG, ParserUtility.dateTimeUtcToString(this.twinMetadata.getLastUpdated()));
             jsonMetadata.addProperty(TwinMetadata.LAST_UPDATE_VERSION_TAG, this.twinMetadata.getLastUpdatedVersion());
+            if(this.twinMetadata.getLastUpdatedBy() != null)
```

Comment:
space after `if`

Reply:
I did add a space here to keep it consistent with other places.

Reply:
I don't see that a space got added.